core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -667,15 +671,55 @@ public boolean hasNext() {
          * @throws java.util.NoSuchElementException if the iteration has no more elements
          */
         @Override
-        public Boolean next() {
+        public boolean nextBoolean() {
             byte b = iterator.nextByte();
             if (b == (byte) 0) {
                 return false;
             }
             if (b == (byte) 1) {
                 return true;
             }
-            return null;
```

Comment:
This breaks missing value handling for BooleanColumns, as a missing value becomes false when you iterate.  That is why the original iterator uses Boolean instead of boolean.  My concern here is that if the iterator were used to copy data between columns, for example, what is missing in the source would become false in the destination.

Reply:
I missed the point of missing values, probably because I used the set method, which does not support setting a missing value (or a byte), whereas append does. Would it be right to add corresponding set methods, i.e. set(int, byte) and set(int, Boolean)?

Reply:
If Column was parameterised with the element type, we could in general support append and set methods that convert null to the missing value. Se issue
 #285